internal/kafka/internal/clusters/ocm_provider.go

```diff
@@ -26,6 +26,14 @@ const (
 	ocmMultiAZClusterNodeScalingMultiple = 3
 )
 
+var (
+	// OCM quota filters
+	QuotaAddonResourceNamePrefix        = "addon"
```

Comment:
Can we namespace the names to include `OCM` or `AMS` to make it more clear that are quota concepts tied to that? ,a s opposed to the general quota concept in kas fleet manager or quota management list